<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="MirrorSettings" name="mirrorWidget"/>
      <widget class="ComponentPage" name="componentWidget"/>
      <widget class="SavePage" name="saveWidget"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComponentPage</class>
   <extends>QWidget</extends>
   <header location="global">componentpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SavePage</class>
   <extends>QWidget</extends>
   <header location="global">savepage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MirrorSettings</class>
   <extends>QWidget</extends>
   <header location="global">mirrorsettings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
